tests/e2e/specs/web-terminal/Readme.md

```diff
@@ -0,0 +1,53 @@
+# Test Scenario for Web Terminal in OpenShift
+
+### Prerequisites:
+* Test Environment: Node.js + Webdriver + Mocha framework.
+* Automation Requirement: A browser for interacting with the OpenShift Web Console.
+* Permissions: Admin access to the test cluster.
+* Setup: Ensure the DevWorkspace Operator is preinstalled before executing the tests.
+## Scenario for WebTerminal under Admin user
+#### Note! This scenario also is used for interop testing
+1. Login as Admin:
+*  Login as admin user into OpenShift Web Console and refresh the page
+2. Clean Up Existing Resources:
+* Remove any existing DevWorkspace custom resources (`oc delete dw --all -n dedicated_namespace`)
+3. Launch WebTerminal:
+*  Click on the "WebTerminal" icon
+* Wait for the WebTerminal widget to appear
+4. Create a WebTerminal Session:
+* Click the "Create" button.
+* Wait for the WebTerminal xterm text area to appear.
+
+5. Verify Admin Session:
+* Execute the command oc whoami and verify that the session is running under the admin user.
+* Since the terminal is canvas-based and difficult to read with WebDriver, read the DevWorkspace object, parse the YAML, and extract the username.
+6. Execute Help Command:
+* Run the help command in the Terminal, redirect the output to a dedicated container, and verify the result.
+* Ensure all tools are present, and the help command executes successfully.
+ 7. Set Inactivity Timeout:
+* Configure the Terminal to have a 30-second inactivity timeout.
```

Comment:
Do you think it would be helpful for testing if you add the command to configure 30-second inactivity timeout?

Reply:
The main goal of the document - a short description of the command presets in the test code: https://github.com/eclipse-che/che/blob/0059eaad63412310baf53e93c354a928541a7af3/tests/e2e/specs/web-terminal/WebTerminalUnderAdmin.spec.ts#L94
Also this one can be found in `help` terminal command and official docs: https://developers.redhat.com/articles/2024/09/02/exploring-full-capabilities-web-terminal-operator?source=sso#customizing_the_web_terminal_with_wtoctl